Resources/views/Collector/swiftmailer.html.twig

```diff
@@ -1,97 +1,182 @@
+                <span class="sf-toolbar-status">{{ collector.messageCount }}</span>
+            {% else %}
+                {{ include('@Swiftmailer/Collector/icon.svg') }}
+                <span class="sf-toolbar-value">{{ collector.messageCount }}</span>
+            {% endif %}
         {% endset %}
+
         {% set text %}
             <div class="sf-toolbar-info-piece">
-                <b>Messages</b>
-                <span>{{ collector.messageCount }}</span>
+                <b>Sent messages</b>
+                <span class="sf-toolbar-status">{{ collector.messageCount }}</span>
             </div>
-            {% for name in collector.mailers %}
-                <hr>
-                <div class="sf-toolbar-info-piece">
-                    <b>{{ name }}</b>
-                    <span>{{ collector.messageCount(name) }}</span>
-                </div>
-                <div class="sf-toolbar-info-piece">
-                    <b>Is spooled ?</b>
-                    <span>{{ collector.isSpool(name) ? 'yes' : 'no' }}</span>
-                </div>
-            {% endfor %}
 
+            {% if profiler_markup_version == 1 %}
+                {% for name in collector.mailers %}
+                    <div class="sf-toolbar-info-piece">
+                        <b>{{ name }}</b>
+                        <span>{{ collector.messageCount(name) }}</span>
+                    </div>
+                    <div class="sf-toolbar-info-piece">
+                        <b>Is spooled?</b>
+                        <span>{{ collector.isSpool(name) ? 'yes' : 'no' }}</span>
+                    </div>
+
+                    {% if not loop.first %}
```

Comment:
are you sure about it ? Shouldn't it be either skipped for the last element instead, or displayed before the content for all except the first element ?